packages/pages/src/common/src/template/paths.ts

```diff
@@ -19,6 +19,7 @@ export const getRelativePrefixToRootFromPath = (path: string): string => {
 /**
  * Converts any path to a posix path delimited by "/". Useful for ensuring that a path will be dynamically importable as only posix-style
  * paths are supported with import.
+ * @public
```

Comment:
Only things we export for users should be marked @public. So not this function since it's internal only.

Reply:
Got it. api-extractor errors if it doesn't have one of the following: public / internal / beta / alpha

Reply:
Is that a new requirement? When I had it enabled in the past I never had to mark internal things specifically. Is it because the file is exported or something?

Reply:
I think b/c it's exported since only a couple functions were causing the errors.